go/cmd/vtctldclient/command/vreplication/materialize/materialize.go

```diff
@@ -47,11 +47,12 @@ func registerCommands(root *cobra.Command) {
 	create.Flags().StringVar(&createOptions.SourceKeyspace, "source-keyspace", "", "Keyspace where the tables queried in the 'source_expression' values within table-settings live.")
 	create.MarkFlagRequired("source-keyspace")
 	create.Flags().Var(&createOptions.TableSettings, "table-settings", "A JSON array defining what tables to materialize using what select statements. See the --help output for more details.")
-	create.MarkFlagRequired("table-settings")
 	create.Flags().BoolVar(&common.CreateOptions.StopAfterCopy, "stop-after-copy", false, "Stop the workflow after it's finished copying the existing rows and before it starts replicating changes.")
 	create.Flags().StringVar(&common.CreateOptions.MySQLServerVersion, "mysql_server_version", fmt.Sprintf("%s-Vitess", config.DefaultMySQLVersion), "Configure the MySQL version to use for example for the parser.")
 	create.Flags().IntVar(&common.CreateOptions.TruncateUILen, "sql-max-length-ui", 512, "truncate queries in debug UIs to the given length (default 512)")
 	create.Flags().IntVar(&common.CreateOptions.TruncateErrLen, "sql-max-length-errors", 0, "truncate queries in error logs to the given length (default unlimited)")
+	create.Flags().BoolVar(&common.CreateOptions.IsReference, "reference", false, "For specifying materialize of reference tables")
```

Comment:
I think it's worth a brief explanation here as the concept is only easily found in the docs today if you already know what you're looking for.
```suggestion
	create.Flags().BoolVar(&common.CreateOptions.IsReference, "reference", false, "Specify this when materializing reference tables: tables that exist in full on every target shard.")
```